tests/python/kaolin/io/test_modelnet.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+import os
+
+import pytest
+import torch
+
+from kaolin.io.off import return_type
+from kaolin.io.modelnet import ModelNet
+
+MODELNET_PATH = '/data/ModelNet'
+MODELNET_TEST_CATEGORY_LABELS = ['bathtub']
+MODELNET_TEST_CATEGORY_LABELS_2 = ['desk']
+MODELNET_TEST_CATEGORY_LABELS_MULTI = ['bathtub', 'desk']
+
+ALL_CATEGORIES = [
+    MODELNET_TEST_CATEGORY_LABELS,
+    MODELNET_TEST_CATEGORY_LABELS_2,
+    MODELNET_TEST_CATEGORY_LABELS_MULTI,
+]
+
+# Skip test in a CI environment
+@pytest.mark.skipif(os.getenv('CI') == 'true' or os.getenv('CI'), reason="CI does not have dataset")
```

Comment:
I believe you just need `os.getenv('CI') == 'true'` now

Reply:
fixed.